charts/karpenter/values.yaml

```diff
@@ -166,10 +166,14 @@ settings:
   batchIdleDuration: 1s
   # -- AWS-specific configuration values
   aws:
+    # -- Role to assume for calling AWS services.
+    assumeRole: ""
```

Comment:
I'd call this `assumeRoleArn` or `assumeRoleARN`. Unless I've misunderstood and it takes a role name.

Reply:
Renamed `assumeRole` to `assumeRoleARN` in next revision.